api-reference/20 Data Visualization Widgets/dxPolarChart/1 Configuration/annotations/annotations.md

```diff
@@ -5,7 +5,191 @@ inherits: dxPolarChart.Options.commonAnnotationSettings
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            Dx{WidgetName}Module
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <Dx{WidgetName} ... >
+            <DxAnnotation
+                type="text"
+                text="Annotation text"
+            />
+            <DxAnnotation
+                type="image"
+                image="http://image/url/myimage.png"
+            />
+        </Dx{WidgetName}>
+    </template>
+
+    <script>
+    import Dx{WidgetName}, {
+        DxAnnotation
+    } from 'devextreme-vue/{widget-name}';
+
+    export default {
+        components: {
+            Dx{WidgetName},
+            DxAnnotation
+        },
+        data() {
+            // ...
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import {WidgetName}, {
+        Annotation
+    } from 'devextreme-react/{widget-name}';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <{WidgetName} ... >
+                    <Annotation
+                        type="text"
+                        text="Annotation text"
+                    />
+                    <Annotation
+                        type="image"
+                        image="http://image/url/myimage.png"
+                    />
+                </{WidgetName}>
+            );
+        }
+    }
+
+    export default App;
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Annotations can be unattached or anchored to a PolarChart element. The following list shows how to position them. PolarChart coordinates ([argument](/api-reference/_hidden/dxChartCommonAnnotationConfig/argument.md '{basewidgetpath}/Configuration/annotations/#argument'), [value](/api-reference/_hidden/dxChartCommonAnnotationConfig/value.md '{basewidgetpath}/Configuration/annotations/#value'), [axis](/api-reference/_hidden/dxChartCommonAnnotationConfig/axis.md '{basewidgetpath}/Configuration/annotations/#axis'), [series](/api-reference/_hidden/dxPolarChartCommonAnnotationConfig/series.md '{basewidgetpath}/Configuration/annotations/#series')), [angle](/api-reference/_hidden/dxPolarChartCommonAnnotationConfig/angle.md '{basewidgetpath}/Configuration/annotations/#angle') and [radius](/api-reference/_hidden/dxPolarChartCommonAnnotationConfig/radius.md '{basewidgetpath}/Configuration/annotations/#radius') specify the element that the annotation's arrow points to; pixel coordinates ([x](/api-reference/_hidden/dxChartCommonAnnotationConfig/x.md '{basewidgetpath}/Configuration/annotations/#x') and [y](/api-reference/_hidden/dxChartCommonAnnotationConfig/y.md '{basewidgetpath}/Configuration/annotations/#y')) specify the position of the annotation's center.
```

Comment:
Angle and radius do not specify an element.
I believe that `axis` doesn't make sense for the PolarChart.